bpf/process/types/kprobes.h

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Tetragon */
+
+#ifndef __LINUX_KPROBES_H__
+#define __LINUX_KPROBES_H__
+
+#ifndef KSYM_NAME_LEN
+#define KSYM_NAME_LEN 128U
+#endif
+
+struct msg_kprobe {
+	u64 addr;
+	u32 offset;
+	u32 pad;
+	char symbol[KSYM_NAME_LEN];
```

Comment:
is it necessary to send the symbol? I'd think that the ksyms.KernelSymbols/kernelSymbols.GetFnOffset on user side that you already use should be fast enough